test/k8sT/Policies.go

```diff
@@ -524,7 +524,18 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 		})
 
 		waitforPods := func() {
-			pods, err := kubectl.WaitforPods(
+			port := "6379"
```

Comment:
Nit: Can we not hardcode this, and declare a var instead?

Reply:
I prefer keeping this variable scoped to this function because `WaitForServiceEndpoints` takes a port as an argument, while later in the test, the port that's returned is what is stored in Kubernetes vs. some hardcoded value. Keeping them separate is best IMO.